### PR TITLE
fix: редактирование текстовых ячеек

### PR DESCRIPTION
**Pull request by Kupov**

### CODE REVIEW

**Comment by Kupov** [comment]
@dontuse

File: dist/assets-table.bundle.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "bd6dc3c730ed9828bab5"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "9e4f3ff954a845d935ff"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-table.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

File: dist/app.bundle.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "bd6dc3c730ed9828bab5"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "9e4f3ff954a845d935ff"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/app.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Too many errors. (0% scanned).

File: src/Table/e-table.scss [hunk 3]
```diff
@@ -35,7 +36,21 @@
...
+      #{$root} {
+        &-cell-text.is-edit {
+          right: 0; left: auto;
+        }
+      }
+    }
+    &:first-child {
+      #{$root} {
+        &-cell-text.is-edit {
+          right: auto; left: 0;
```
Comment (dontuse) [inline src/Table/e-table.scss]: на этой же строке, нет же вложений

**Comment by Kupov** [comment]
:up:

**Pull request merged**